File: src/Web/CombatAnalysis.EnhancedWebApp/CombatAnalysis.EnhancedWebApp.Server/Models/WoWGameData/Character/MythicKeystone/MythicKeystoneCurrentPeriodModel.cs
using System.Text.Json.Serialization;

namespace CombatAnalysis.EnhancedWebApp.Server.Models.WoWGameData.Character.MythicKeystone;

public class MythicKeystoneCurrentPeriodModel
{
    [JsonPropertyName("period")]
    public MythicKeystoneSeasonModel Period { get; set; }

    [JsonPropertyName("best_runs")]
    public MythicKeystoneBestRunModel[] BestRuns { get; set; }
}

File: src/Web/CombatAnalysis.EnhancedWebApp/combatanalysis.enhancedwebapp.client/src/features/wowGameData/types/mythicKeystone/MythicKeystoneCurrentPeriodModel.ts
import type { MythicKeystoneBestRunModel } from './MythicKeystoneBestRunModel';
import type { MythicKeystoneSeasonModel } from './MythicKeystoneSeasonModel';

export type MythicKeystoneCurrentPeriodModel = {
    period: MythicKeystoneSeasonModel;
    bestRuns: MythicKeystoneBestRunModel[];
}